WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -177,4 +177,12 @@ def duration_to_s(total_seconds)
 
     [hours > 0 ? "#{hours}h " : '', minutes > 0 ? "#{minutes}m " : '', format('%.2f', seconds), 's'].join
   end
+
+  def wca_id_link(wca_id, options = {})
+    if wca_id
+      content_tag :span, class: "wca-id" do
+        link_to wca_id, "#{root_url}results/p.php?i=#{wca_id}", options
```

Comment:
This seems like a king of breaking your anti-principal *You just have to remember*, i.e. if someone grabs (c&p) the helper call from a view and modifies it, we have to catch that during review, so that the `relative: true` part is not present in mailers as you said. Not sure whether or not we save enough space to take this risk, but if you still consider it worth doing (?) I'll follow your experience =)